ui/helpers/utils/util.test.js

```diff
@@ -301,4 +301,31 @@ describe('util', () => {
       });
     });
   });
+
+  describe('toHumanReadableTime()', () => {
+    const t = (key, number) => {
+      switch (key) {
+        case 'gasTimingSecondsShort':
+          return `${number} sec`;
+        case 'gasTimingMinutesShort':
+          return `${number} min`;
+        case 'gasTimingHoursShort':
+          return `${number} hrs`;
+        default:
+          return '';
+      }
+    };
+    it('should return empty string if milliseconds passed is undefined', () => {
+      expect(util.toHumanReadableTime(t)).toStrictEqual('');
+    });
+    it('should return value in seconds for milliseconds passed is less than 9000, that is 90 seconds', () => {
+      expect(util.toHumanReadableTime(t, 6000)).toStrictEqual('6 sec');
```

Comment:
it will show as 0 seconds, this is existing logic used to show transaction time, which I moved to this util - thus I guess < 1 sec is not considered significant.

Reply:
It looks like based on the code that any sub-milliseconds will get rounded up, so 6300 will get shown as "7 sec", right?

Reply:
i18n code actually handles rounding part, so I need to check, let me do that and add a test case for this

Reply:
Also added test case for this

Reply:
I appreciate it!